cmd/plume/upload.go

```diff
@@ -126,17 +200,20 @@ func getImageVersion(imagePath string) string {
 	return version
 }
 
-func writeFile(client *http.Client, filename string) error {
+// Write file to Google Storage
+func writeFile(client *http.Client, filename, destname, projec, projectID string) error {
 	ctx := cloud.NewContext(projectID, client)
```

Comment:
Why is projectID needed? The upload API only needs a project ID to create a new bucket. Is this a quirk of the 'cloud' library? All of my work has been on the direct API bindings.